__tests__/app.test.js

```diff
@@ -326,29 +326,48 @@ describe("PATCH /api/articles/:article_id", () => {
   });
 });
 
-describe('DELETE /api/comments/:comment_id', () => {
-  it('Responds with 204 and no content when a comment is successfully deleted', () => {
+describe("DELETE /api/comments/:comment_id", () => {
+  it("Responds with 204 and no content when a comment is successfully deleted", () => {
     return request(app)
-      .delete('/api/comments/1')
+      .delete("/api/comments/1")
       .expect(204)
       .then((response) => {
         expect(response.body).toEqual({});
       });
   });
-  it('Responds with 404 for non-existent comment', () => {
+  it("Responds with 404 for non-existent comment", () => {
     return request(app)
-      .delete('/api/comments/9999')
+      .delete("/api/comments/9999")
       .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toBe('comment not found');
+        expect(body.msg).toBe("comment not found");
       });
   });
-  it('Responds with 400 for invalid comment ID', () => {
+  it("Responds with 400 for invalid comment ID", () => {
     return request(app)
-      .delete('/api/comments/NaN')
+      .delete("/api/comments/NaN")
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe('invalid input');
+        expect(body.msg).toBe("invalid input");
+      });
+  });
+});
+
+describe("GET /api/users", () => {
+  it("Responds with an array of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        expect(Array.isArray(users)).toBe(true);
+        expect(users.length).toBeGreaterThan(0);
```

Comment:
You can be more specific here. According to the test data, how many user do you expect to get back?